src/js/containers/dashboard/DashboardContentContainer.jsx

```diff
@@ -0,0 +1,29 @@
+/**
+ * DashboardContentContainer.jsx
```

Comment:
We'll may want to coordinate with @dpb-bah on where he's putting the Submission Summary section. I'm also thinking this could be just a component (that gets passed the active filters as a prop) and displays containers for each visualization.